clang/lib/Driver/ToolChains/MSVC.cpp

```diff
@@ -689,6 +689,42 @@ void MSVCToolChain::AddSystemIncludeWithSubfolder(
   addSystemInclude(DriverArgs, CC1Args, path);
 }
 
+#ifdef ENABLE_CLASSIC_FLANG
+/// Convert path list to Fortran frontend argument
+static void AddFlangSysIncludeArg(const ArgList &DriverArgs,
+                                  ArgStringList &Flang1Args,
+                                  ToolChain::path_list IncludePathList) {
+  std::string ArgValue; // Path argument value
+
+  // Make up argument value consisting of paths separated by colons
+  bool first = true;
+  for (auto P : IncludePathList) {
+    if (first) {
+      first = false;
+    } else {
+      ArgValue += ":";
```

Comment:
using `:` as a separator in the Windows world seems tricky, how would it look like? `"c:\msvc:d:\projects"`, is this approach ok?

Reply:
No, officially the separator is the semicolon ';' on Windows. Thanks for the review. Switching between the terminals, (mingw, ms command prompt) confuses me sometimes.